### ISSUE TITLE
Data validation utils.

### ISSUE THREAD

**Issue by raw-leak** [MykhayloGusak/rest_api_middleware_assessment#13]
Module to validate if data is required, type string, number and/or integer. resolve #14 

Join validation is nices to be inspired by making the interface.

```
import { validate } = require("...")

someFunction = (string, number, integer) =>{
try{
   validate(param).required().string()
   validate(number).required().number()
   validate(integer).required().integer()  or   validate(integer).required()number().integer() 
  ...
} catch (err){
  ...
}
}
```

**Comment by raw-leak** [comment]
Already done. 

This closes #14 .

**Comment by MykhayloGusak** [comment]
This fixes #14

**Comment by raw-leak** [comment]
This resolved #13
